diff --git a/node_modules/@remix-run/testing/dist/index.d.ts b/node_modules/@remix-run/testing/dist/index.d.ts
index fe7a155..39eb894 100644
--- a/node_modules/@remix-run/testing/dist/index.d.ts
+++ b/node_modules/@remix-run/testing/dist/index.d.ts
@@ -1,2 +1,2 @@
 export type { RemixStubProps } from "./create-remix-stub";
-export { createRemixStub as unstable_createRemixStub } from "./create-remix-stub";
+export { createRemixStub } from "./create-remix-stub";
diff --git a/node_modules/@remix-run/testing/dist/index.js b/node_modules/@remix-run/testing/dist/index.js
index 8ace74a..aec3dba 100644
--- a/node_modules/@remix-run/testing/dist/index.js
+++ b/node_modules/@remix-run/testing/dist/index.js
@@ -16,4 +16,4 @@ var createRemixStub = require('./create-remix-stub.js');
 
 
 
-exports.unstable_createRemixStub = createRemixStub.createRemixStub;
+exports.createRemixStub = createRemixStub.createRemixStub;
