diff --git a/node_modules/@testing-library/react/types/index.d.ts b/node_modules/@testing-library/react/types/index.d.ts
index 558edfa..70b4b7d 100644
--- a/node_modules/@testing-library/react/types/index.d.ts
+++ b/node_modules/@testing-library/react/types/index.d.ts
@@ -92,11 +92,11 @@ export function render<
 >(
   ui: React.ReactElement,
   options: RenderOptions<Q, Container, BaseElement>,
-): RenderResult<Q, Container, BaseElement>
+): Promise<RenderResult<Q, Container, BaseElement>>
 export function render(
   ui: React.ReactElement,
   options?: Omit<RenderOptions, 'queries'>,
-): RenderResult
+): Promise<RenderResult>
 
 export interface RenderHookResult<Result, Props> {
   /**
@@ -146,7 +146,7 @@ export function renderHook<
 >(
   render: (initialProps: Props) => Result,
   options?: RenderHookOptions<Props, Q, Container, BaseElement>,
-): RenderHookResult<Result, Props>
+): Promise<RenderHookResult<Result, Props>>
 
 /**
  * Unmounts React trees that were mounted with render.
@@ -158,6 +158,4 @@ export function cleanup(): void
  * If that's not available (older version of react) then it
  * simply calls the given callback immediately
  */
-export const act: typeof reactAct extends undefined
-  ? (callback: () => void) => void
-  : typeof reactAct
+export const act: (callback: () => void) => Promise<void>
