diff --git a/node_modules/@testing-library/react/types/index.d.ts b/node_modules/@testing-library/react/types/index.d.ts
index 37c8392..dcdbbab 100644
--- a/node_modules/@testing-library/react/types/index.d.ts
+++ b/node_modules/@testing-library/react/types/index.d.ts
@@ -145,11 +145,11 @@ export function render<
 >(
   ui: React.ReactNode,
   options: RenderOptions<Q, Container, BaseElement>,
-): RenderResult<Q, Container, BaseElement>
+): Promise<RenderResult<Q, Container, BaseElement>>
 export function render(
   ui: React.ReactNode,
   options?: Omit<RenderOptions, 'queries'>,
-): RenderResult
+): Promise<RenderResult>
 
 export interface RenderHookResult<Result, Props> {
   /**
@@ -239,7 +239,7 @@ export function renderHook<
 >(
   render: (initialProps: Props) => Result,
   options?: RenderHookOptions<Props, Q, Container, BaseElement>,
-): RenderHookResult<Result, Props>
+): Promise<RenderHookResult<Result, Props>>
 
 /**
  * Unmounts React trees that were mounted with render.
@@ -252,6 +252,4 @@ export function cleanup(): void
  * simply calls the deprecated version which is ReactTestUtils.act(cb)
  */
 // IfAny<typeof reactAct, reactDeprecatedAct, reactAct> from https://stackoverflow.com/a/61626123/3406963
-export const act: 0 extends 1 & typeof reactAct
-  ? typeof reactDeprecatedAct
-  : typeof reactAct
+export const act: (callback: () => void) => Promise<void>
